README.md

```diff
@@ -631,6 +623,14 @@ Thanks goes to these wonderful peoples :
   </tr>
```

Comment:
Please use Grammerly or any relative tools for sentence correction

Reply:
resolved the comments